src/net_processing.cpp

```diff
@@ -953,7 +953,7 @@ void PeerManager::FinalizeNode(NodeId nodeid, bool& fUpdateConnectionTime) {
     LogPrint(BCLog::NET, "Cleared nodestate for peer=%d\n", nodeid);
 }
 
-bool GetNodeStateStats(NodeId nodeid, CNodeStateStats &stats) {
+bool PeerManager::GetNodeStateStats(NodeId nodeid, CNodeStateStats &stats) {
```

Comment:
It's a bit confusing to have `GetNodeStateStats` which is actually a member of `PeerManager`, and still have a `CNodeState`, even if inside this function you effectively fetch it. Maybe rename `GetNodeProcessingStats` ?

Reply:
I think the name right now is fine. It's getting stats related to the node's state.